workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/controller/WorkflowsMgtApiController.java

```diff
@@ -35,12 +33,7 @@ public class WorkflowsMgtApiController implements WorkflowsMgtApi {
 
   @Override
   @Authorized(headerTokenKey = X_MANAGEMENT_TOKEN_KEY)
-  public ResponseEntity<Void> saveAndDeploySwadl(String token, SwadlView swadlView, HttpServletRequest request) {
-    Object userAttribute = request.getAttribute("user");
-    if (userAttribute != null) {
-      UserClaim user = (UserClaim) userAttribute;
-      swadlView.setCreatedBy(user.getId());
-    }
```

Comment:
How do we set the creator now?

Reply:
in the createdBy field from the the request. because sometimes, we need to reassign the owner whenever it is necessary